vagrant/Makefile

```diff
@@ -35,6 +35,9 @@ setup.clients:
 generate.report:
 	@$(VSSH) admin "${SSH_EXTRA_VARS} make -C /home/vagrant/ansible generate.report"
 
+nodes.statedump:
+	@ansible-playbook --inventory $(INVENTORY) ${ANSIBLE_EXTRA_VARS} nodes.statedump.yml
```

Comment:
I guess this will be called from jenkins after finishing the job (either successfully or with errors), right ?

Shouldn't we also add an entry in the main Makefile ?

Reply:
> I guess this will be called from jenkins after finishing the job (either successfully or with errors), right ?

Yes, that's the plan.

> Shouldn't we also add an entry in the main Makefile ?

Yes, thanks for the reminder.
Added now.